.travis.yml

```diff
@@ -3,7 +3,6 @@ language: go
 go_import_path: go.uber.org/fx
 
 go:
-  - 1.7
```

Comment:
Sure, no problem. The other option is to not close the server on app close and leave a TODO to add that once all our supported Go versions have a `Close` or `Shutdown`.

I'd prefer to make the example realistic, since we don't have any users on Go 1.7.